legislators-social-media.yaml

```diff
@@ -5450,3 +5451,46 @@
     govtrack: 412704
   social:
     facebook: rogermarshallmd
+    twitter: RepMarshall
+- id:
+    bioguide: E000296
+    govtrack: 412677
+  social:
+    twitter: RepDwightEvans
+- id:
+    bioguide: R000609
+    govtrack: 412692
+  social:
+    facebook: RepRutherfordFL
+    twitter: RepRutherfordFL
```

Comment:
This is linked to from https://rutherford.house.gov, and it comes up now when I run our script. I guess it was added in the last week. I'll do another sweep and see if other stuff comes up.